DatadogSessionReplay/Sources/SessionReplayConfiguration.swift

```diff
@@ -49,10 +54,12 @@ extension SessionReplay {
         public init(
             replaySampleRate: Float,
             defaultPrivacyLevel: SessionReplayPrivacyLevel = .mask,
+            startRecordingImmediately: Bool = true,
```

Comment:
nit/ We should also list it within `/// - Parameters:` list in this API comment.

Reply:
Totally! That's a miss from my side when making the changes. Good catch.